Misc/NEWS.d/next/Core_and_Builtins/2024-12-07-13-06-09.gh-issue-127599.tXCZb_.rst

```diff
@@ -0,0 +1,4 @@
+Fix statistics for increments of object reference counts (in particular, when
+a reference count was increased by more than 1 in a single operation).
```

Comment:
We can keep this one and drop the sentence about _Py_RefcntAdd.

Reply:
Thanks, done.